puzzle-utils/encoders/caesar.py

```diff
@@ -14,22 +14,54 @@ def shift_letter(letter, shift_count):
     return chr(char_code + CHARACTER_OFFSET)
 
 def caesar(word, shift_count):
+    """
+    Caesar shift a whole word by a number
+
+    Params:
+    word        (required) - The word to shift
+    shift_count (required) - The count to shift by
+
+    Alternatives:
+    brute_caesar       - Shift a whole word by 1 - 25
+    independent_caesar - Shift a word by different numbers for each letter
+    """
     new_word = ''
     for letter in word.upper():
         new_word += shift_letter(letter, shift_count)
     return new_word
 
 def brute_caesar(word):
```

Comment:
Not documentation related but I'd prefer this function and the one below was inverted (e.g. caesar_brute) so that it pseudo namespaces the functions. What prompted this was that I tried to use tab to complete to see available functions for caesar only the basic one was in the results.